src/ray/raylet/node_manager.cc

```diff
@@ -552,9 +553,34 @@ void NodeManager::NodeRemoved(const GcsNodeInfo &node_info) {
   // node.
   rpc::Address address;
   address.set_raylet_id(node_info.node_id());
+  RAY_LOG(WARNING) << "HandleUnexpectedWorkerFailure 1.0";
   HandleUnexpectedWorkerFailure(address);
 }
 
+// XXX
+std::string NodeManager::PrintLeasedWorkers() {
```

Comment:
Please remove these. If you really need the print messages, pass the state that you need to check (e.g., `leased_workers_`) into an anonymous helper function instead of making it a method on the `NodeManager`.